### PR TITLE
Replaced pkg/errors with errors

### PR DESCRIPTION
**Pull request by briancaine**
https://github.com/skycoin/exchange-api/issues/2

I think `pkg/errors` includes stack traces while stdlib `errors` is basically just a string value.

`errors.New` is the same in either package. If you `import "fmt"`, `fmt.Errorf` should be equivalent to the `pkg/errors` version of `errors.Errorf`.

The `pkg/errors` `errors.Wrapf` function wraps an existing error with a fresh error. So I just remove the error it's trying to wrap and replaced those calls with `fmt.Errorf`.

### CODE REVIEW

**Comment by gz-c** [comment]
There were no calls to `Wrapf` so it wasn't necessary for them to use `github.com/pkg/errors`

**Comment by gz-c** [comment]
Need to fix merge conflicts

**Comment by briancaine** [comment]
@gz-c There were a few calls to `errors.Wrapf`. See [this PR commit](https://github.com/skycoin/exchange-api/pull/18/commits/47a6ab665c4ae42d2bedc902f272648a8fb063d1#diff-3b21b9db07553752c57ca06f0fc70742L222).

File: exchange/cryptopia.co.nz/api.go [hunk 1]
```diff
@@ -219,10 +220,10 @@ func getMarketOrderGroups(count int, markets ...string) ([]MarketOrdersWithLabel
 	}
 	resp, err := requestGet("getmarketordergroups", requestParams)
 	if err != nil {
-		return nil, errors.Wrapf(err, "GetMarketOrderGroups failed, markets: %s", strings.Join(markets, " "))
+		return nil, fmt.Errorf("GetMarketOrderGroups failed, markets: %s", strings.Join(markets, " "))
```
Comment (gz-c) [inline exchange/cryptopia.co.nz/api.go]: Put the original error message into the format string for all of the formerly Wrapf calls

**Review by gz-c — CHANGES_REQUESTED** [review]

File: exchange/cryptopia.co.nz/api.go (hunk 1 above)
Comment (briancaine) [reply exchange/cryptopia.co.nz/api.go]: https://github.com/skycoin/exchange-api/pull/18/commits/8c61cc882a298c312243df8b71e1df211c345c8b

Done.

**Comment by briancaine** [comment]
Done.

**Pull request merged**